app/routes/auth_routes.py

```
@@ -12,6 +12,14 @@
 from app.utils.auth_utils import check_password, generate_token, hash_password
 
 bp = Blueprint('auth', __name__, url_prefix='/auth')
+def generate_token(user_id, role):
```

Ruff (0.8.2)

Redefinition of unused `generate_token` from line 12

(F811)